### PR TITLE
 User Defined Body Propterties v2

### PR DESCRIPTION
**Pull request by MShabara**
Replaced the PR #1335 
Allows the user to change the name of the hydro bodies.
Allows printing the non-hydro bodies data in the command window.

### CODE REVIEW

File: source/functions/initializeWecSim.m [hunk 1]
```diff
@@ -511,11 +511,25 @@
...
     waves(iW).listInfo();
 end; clear iW
-fprintf('\nList of Body: ');
-fprintf('Number of Bodies = %u \n',simu.numHydroBodies)
-for i = 1:simu.numHydroBodies
-    body(i).listInfo
+fprintf('\nList of Body:\n ');
+fprintf('Number of Hydro Bodies = %u \n',simu.numHydroBodies)
+i = 1; 
+while i <= simu.numHydroBodies
```
Comment (jtgrasb) [inline source/functions/initializeWecSim.m:517]: To be consistent with the rest of the source code, I think we should use for loops. "for i = 1:simu.numHydroBodies"

**Comment by akeeste** [comment]
FYI @jtgrasb @MShabara all can ignore my two test commits. They did not end up changing anything. I was testing a github issue and mshabara's is the only repo I don't currently have collaborator access to, so I used this PR.

File: source/functions/initializeWecSim.m (hunk 1 above)
Comment (MShabara) [reply source/functions/initializeWecSim.m]: I applied your recommendations.
I also made the code print the non-hydro body details same as the hydro body

**Comment by jtgrasb** [comment]
@MShabara Thanks for the updates. The last thing is to discard the changes to `rm3.h5` from the PR, then it will be ready for merge.

**Comment by MShabara** [comment]
I discarded the changes to rm3.h5 from the PR. If things look good to you, feel free to merge this PR.

**Pull request merged**